package.json

```diff
@@ -42,48 +42,48 @@
     "README.md"
   ],
   "devDependencies": {
-    "@commitlint/cli": "^11.0.0",
-    "@commitlint/config-conventional": "^11.0.0",
-    "@nestjs/common": "^8.0.0",
-    "@nestjs/core": "^8.0.0",
-    "@nestjs/testing": "^8.0.0",
-    "@pact-foundation/pact": "^9.0.0",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
+    "@commitlint/cli": "^17.1.2",
+    "@commitlint/config-conventional": "^17.1.0",
+    "@nestjs/common": "^9.1.4",
+    "@nestjs/core": "^9.1.4",
+    "@nestjs/testing": "^9.1.4",
+    "@pact-foundation/pact": "^10.1.4",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
     "@types/get-port": "^4.2.0",
-    "@types/jest": "24.9.1",
-    "@typescript-eslint/eslint-plugin": "^2.33.0",
-    "@typescript-eslint/parser": "^2.33.0",
-    "commitizen": "^4.2.2",
+    "@types/jest": "29.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.40.1",
+    "@typescript-eslint/parser": "^5.40.1",
+    "commitizen": "^4.2.5",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^7.15.0",
-    "eslint-config-prettier": "^3.6.0",
-    "eslint-import-resolver-typescript": "^2.0.0",
-    "eslint-plugin-import": "2.20.2",
-    "eslint-plugin-jest": "^23.11.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "husky": "^4.3.5",
-    "jest": "24.9.0",
-    "jest-junit": "12.0.0",
-    "lint-staged": "^10.5.3",
-    "prettier": "^2.2.1",
+    "eslint": "8.25.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-typescript": "^3.5.2",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-jest": "^27.1.3",
+    "eslint-plugin-prettier": "^4.2.1",
+    "husky": "^8.0.1",
+    "jest": "29.2.1",
+    "jest-junit": "14.0.1",
+    "lint-staged": "^13.0.3",
+    "prettier": "^2.7.1",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.0",
-    "semantic-release-cli": "^5.4.0",
-    "ts-jest": "24.1.0",
-    "ts-loader": "^6.2.2",
-    "ts-node": "8.10.2",
-    "tsconfig-paths": "^3.9.0",
+    "semantic-release-cli": "^5.4.4",
+    "ts-jest": "29.0.3",
+    "ts-loader": "^9.4.1",
+    "ts-node": "10.9.1",
+    "tsconfig-paths": "^4.1.0",
     "tslint-config-airbnb": "^5.11.2",
-    "typescript": "^4.0.0"
+    "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "@nestjs/common": "7.x || 8.x",
-    "@nestjs/core": "7.x || 8.x",
-    "@pact-foundation/pact": "^9.0.0"
+    "@nestjs/common": "7.x || 8.x || 9.x",
+    "@nestjs/core": "7.x || 8.x || 9.x",
+    "@pact-foundation/pact": "^10.0.0"
```

Comment:
This is a breaking change and should be indicated in the commit message

Reply:
Do you mean I should amend it? I was hoping to squash merge in the end and add the information in the final commit message. Does this make sense? Or should I reset this branch and redo it?

Reply:
As a general rule, I think squash merging loses information and doesn't encourage good commit messages. Since most Pact-Foundation projects build the changelog from the commit history, it's nicer to merge the actual history. To squash or not is kind of a side point though - I might write this up as a separate blog post in the next couple of weeks.

If you're planning to squash merge, I would update the PR title to indicate the breaking change.